### DISCUSSION TITLE
🧩 FEATURE REQUEST: Context menu is not optimised

### DISCUSSION THREAD

**Discussion by JayrajSingh65** [dailydotdev/daily#1579] [Feature Requests]
### Description

Hey daily dev  I don't know how your

 designer took the decision but it's look bad
 to the user showing so many not interested text  in a bad manner
![Screenshot_2023-03-30-19-31-49-111_com.android.chrome.jpg](https://user-images.githubusercontent.com/97652040/228862198-3c8b5278-ed87-4cbd-aee6-0bdd3709ecd2.jpg)

### Solution

  Rather than showing directly you can show these text in report screen which will improve user experience
That's all.

**Comment by idoshamun** [comment]
Hi @JayrajSingh65,

Thanks for the feedback! We will consider how to fix it.

**Comment by karthiknadar1204** [comment]
If this issue is still open,please assign it to me,i would like to work on it

**Comment by sshanzel** [comment]
> If this issue is still open,please assign it to me,i would like to work on it

Hi! Thank you for your motivation and interest in fixing this concern. Once we get the final decision on how to proceed, we will ping you here.
